_posts/2016-07-07-better-email-validation-with-obey-and-mailgun.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Better Email Validation with Obey and Mailgun"
+date: 2016-07-07 10:00:00
+categories: asynchronous validation nodejs
+author: kent.safranski
+cover: /assets/images/covers/books.jpg
+description: Email validation seems simple, but you really should be using a service to accurately validate addresses. Here's how Obey and Mailgun can work together to reliably validate emails.
+comments: true
+---
+
+Validating email is simple, right? Just throw a regex at it? Well it's not as simple as you might think. The RFC Spec (Page 27 of [this monster](https://www.ietf.org/rfc/rfc0822.txt), if you're interested) is clear as mud on what constitutes a valid email.
+
+Basically, with all the special characters and conditions of what can go before and after the `@`, a regex to validate emails accurately would be incomprehensable at best.
```

Comment:
Stylistically, I'd kill the "Basically, " and move this sentence to the end of the last paragraph. But now that I've typed that, I'm realizing that you probably did it to get a good blurb for the post summary on the index page, so if that's the case, ignore me.